zc_common/remote_resource/mixins.py

```diff
@@ -1,24 +0,0 @@
-"""
```

Comment:
Yes. This was replaced with the generic `?filter[]` system in #25 . We just missed this.